contrib/core-contract-tests/vitest.config.js

```diff
@@ -0,0 +1,36 @@
+/// <reference types="vitest" />
+
+import { defineConfig } from "vite";
+import { vitestSetupFilePath, getClarinetVitestsArgv } from "@hirosystems/clarinet-sdk/vitest";
+
+/*
+  In this file, Vitest is configured so that it works seamlessly with Clarinet and the Simnet.
+
+  The `vitest-environment-clarinet` will initialise the clarinet-sdk
+  and make the `simnet` object available globally in the test files.
+
+  `vitestSetupFilePath` points to a file in the `@hirosystems/clarinet-sdk` package that does two things:
+    - run `before` hooks to initialize the simnet and `after` hooks to collect costs and coverage reports.
```

Comment:
```suggestion
    - run `before` hooks to initialize the Simnet and `after` hooks to collect costs and coverage reports.
```

Reply:
We usually refer to mainnet, testnet, devnet, and simnet with lower case in docs, blog posts and such

Reply:
True, thanks 👍